fuzz/src/chanmon_consistency.rs

```diff
@@ -1722,6 +1661,67 @@ pub fn do_test<Out: Output>(data: &[u8], underlying_out: Out, anchors: bool) {
 				nodes[2].maybe_propose_quiescence(&nodes[1].get_our_node_id(), &chan_b_id).unwrap()
 			},
 
+			0xb0 | 0xb1 | 0xb2 => {
```

Comment:
Moving the match arms, isn't that a noop and not related to bwd compat?

Reply:
"Since we broke backwards compat and changed the values of bytes
for reloading nodes in the `chanmon_consistency` fuzzer, here we
move them down to the right position."

I don't understand why breaking backwards compat needs mentioning in this commit message. I think the word 'since' confuses me?